lua/entities/gmod_wire_expression2/core/table.lua

```diff
@@ -1122,7 +1122,6 @@ registerCallback( "postinit", function()
 			rv1.n[n] = rv2
 			rv1.ntypes[n] = id
 			self.GlobalScope.vclk[rv1] = true
-			return rv2
```

Comment:
Ok, question:
How do you return a default E2 value from the type ID? (Is there even such a function in `E2Lib`?)